src/Crate/PDO/Http/Client.php

```diff
@@ -1,231 +0,0 @@
-<?php
-/**
- * Licensed to CRATE Technology GmbH("Crate") under one or more contributor
- * license agreements.  See the NOTICE file distributed with this work for
- * additional information regarding copyright ownership.  Crate licenses
- * this file to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.  You may
- * obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial agreement.
- */
-
-namespace Crate\PDO\Http;
-
-use Crate\PDO\Exception\RuntimeException;
-use Crate\PDO\Exception\UnsupportedException;
-use Crate\Stdlib\Collection;
-use GuzzleHttp\Exception\BadResponseException;
-use GuzzleHttp\Exception\ConnectException;
-use GuzzleHttp\Exception\ParseException;
-
-class Client implements ClientInterface
```

Comment:
Remember that Bc breakages need to be documented

Reply:
Yeah, we are releasing a new major so I broke a bunch of stuff for the reasons.

Reply:
:+1: